MekHQ/src/mekhq/gui/dialog/PopupValueChoiceDialog.java

```diff
@@ -1,28 +1,31 @@
 /*
- * AlertPopup.java
+ * Copyright (c) 2013 The MegaMek Team. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2010, 2013 The MegaMek Team. All rights reserved.
```
Per:
> `* Created on Jan 6, 2010, 10:46:02 PM`

Reply:
The file is actually 2013, the header was copied and never updated with the date

Reply:
As in the header was copied from another file?

Reply:
Yep